astropy/constants/__init__.py

```diff
@@ -25,9 +25,10 @@
 except ImportError:
     pass
 
-from .constant import Constant, EMConstant
```

Comment:
I think we cannot change this import; people may count on `EMConstant` being available.

Similarly, I think you may need to keep the imports of `si` and `cgs` just so people can do `const.si.c`